src/styles/styles/header.scss

```diff
@@ -0,0 +1,64 @@
+.header {
+  display: flex;
+  box-sizing: border-box;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  padding-left: 50px;
+  padding-right: 50px;
+  align-items: center;
+  justify-content: space-between;
+
+  &__logo-link {
+    line-height: 0;
+  }
+
+  &__list{
+    display: flex;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    list-style: none;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+   display: inline-block;
+   height: 60px;
+   font-weight: 500;
+   font-size: 12px;
+   line-height: 60px;
+   text-align: center;
+   color: $main-color;
+   text-decoration: none;
+   text-transform: uppercase;
+   transition: 0.3s;
+  }
+
+  &__link:hover {
+   color: $button-color;
+  }
+
+  .is-active {
+   position: relative;
+   color:$button-color;
+  }
+
+  .is-active::after {
+   content: "";
+   position: absolute;
+   bottom: 0;
+   display: block;
+   width: 100%;
+   height: 4px;
+   background: $button-color;
+   border-radius: 8px;
+  }
+}
+
+
+
```

Comment:
too mane lines, please delete them and leave just one

good practice is to leave 1 line at the end of the file (please do this fix in each your file)